contracts/SpokePool.sol

```diff
@@ -1012,10 +1216,27 @@ abstract contract SpokePool is
             relayerRefundLeaf.leafId,
             relayerRefundLeaf.l2TokenAddress,
             relayerRefundLeaf.refundAddresses,
+            deferredRefunds,
             msg.sender
         );
     }
 
+    /**
+     * @notice Enables a relayer to claim outstanding repayments. Should virtually never be used, unless for some reason
+     * relayer repayment transfer fails for reasons such as token transfer reverts due to blacklisting. In this case,
+     * the relayer can still call this method and claim the tokens to a new address.
+     * @param l2TokenAddress Address of the L2 token to claim refunds for.
+     * @param refundAddress Address to send the refund to.
+     */
+    function claimRelayerRefund(address l2TokenAddress, address refundAddress) public {
```

Comment:
So I see `refundAddress` is type `address`, and the `relayerRefund` mapping also uses an address=>address key. This is because these variables are only used on the chain that this spoke pool would be deployed to, which will always be an EVM?

Reply:
I think we can migrate this to `bytes32` as well to be consistent with the rest of updates. What do you think @chrismaree?

Reply:
I agree for consistency it should be moved over. doing it now & updating the tests.

Reply:
have updated this accordingly.

Reply:
are there any storage/gas cost considerations to think about with this change?